src/components/link.js

```diff
@@ -0,0 +1,39 @@
+var registerComponent = require('../core/component').registerComponent;
+
+module.exports.Component = registerComponent('link', {
+  schema: {
+    url: { default: '' },
```

Comment:
`link="href: foo.bar; on: hit"` is fine. Separate attribute/component isn't consistent with the framework, prescribing an event is inflexible to different use cases.